addthis.install

```diff
@@ -99,10 +100,16 @@ function addthis_update_7402() {
   variable_del('addthis_widget_load_type');
 }
 
-/**
- * Remove old TWITTER_TEMPLATE_KEY variable, add default value for
- * TWITTER_TEXT_KEY
+/** 
+ * Remove old TWITTER_TEMPLATE_KEY variable
  */
 function addthis_update_7403() {
   variable_del('addthis_twitter_template');
 }
+
+/**
+ * Remove old PLUGIN_INFO_ENABLED_KEY variable.
+ */
+function addthis_update_7404() {
+  variable_set(AddThis::PLUGIN_INFO_ENABLED_KEY, TRUE);
+}
```

Comment:
I would not force existing users to enable this. Only new users should have this as a default together with a notice in the README about this information beeing exposed.